fml/backtrace.h

```diff
@@ -5,14 +5,10 @@
 #ifndef FLUTTER_FML_BACKTRACE_H_
 #define FLUTTER_FML_BACKTRACE_H_
 
-#include <string>
-
 #include "flutter/fml/macros.h"
 
 namespace fml {
 
-std::string BacktraceHere(size_t offset = 0);
```

Comment:
Why remove this function? I presume the intent was for use when debugging.

Reply:
Since the #16450, which introduced `BacktraceHere`, this is only used in signal handler (and test for `BacktraceHere`). I thought it was made public to test.
The handler from `absl::InstallFailureSignalHandler` also shows stacktrace when receive signal, same as from fml's `InstallCrashHandler` before. The stacktrace's format is a little different from BacktraceHere, but it seems that the existing tests doesn't depend on the format.

I just intended to avoid bitrot to follow the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abandonware). I know this can be used to debug. If any engine's contributors use it to debug, I'll restore it.

Reply:
sgtm -- I don't use it to debug, so personally I'm fine if we remove it (we can always restore it in a followup patch with appropriate tweaks if desired)

Reply:
We shouldn't have code that isn't used, but I do end up using this call frequently when debugging/testing locally. Does absl have a similar function?

Reply:
(This is not blocking for landing, but I'd like to know for my own edification)

Reply:
If people are using it for development, and if absl doesn't have an equivalent then let's restore it (the existing test will avoid bitrot, at least to some extent).

Reply:
done